lib/internal/modules/esm/formats.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-const {
-  RegExpPrototypeExec,
-  Uint8Array,
-} = primordials;
+const { RegExpPrototypeExec } = primordials;
 const { getOptionValue } = require('internal/options');
-
-const { closeSync, openSync, readSync } = require('fs');
+const { getValidatedPath } = require('internal/fs/utils');
+const pathModule = require('path');
```

Comment:
We generally call this `path`.

```suggestion
const path = require('path');
```